dagql/session_cache.go

```diff
@@ -59,6 +64,14 @@ func WithTelemetry(telemetry TelemetryFunc) CacheCallOpt {
 	})
 }
 
+// TODO:
+// TODO:
+// TODO:
+// TODO:
```

Comment:
![](https://c.tenor.com/WuOwfnsLcfYAAAAC/star-wars-obi-wan-kenobi.gif)

Reply:
I swear I always ctrl-F `TODO:` and always miss one somehow....